src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
```

Comment:
Using the universal selector (*) is against the checklist requirements due to potential performance impacts. Consider applying styles more selectively to improve performance.